data/companies.json

```diff
@@ -592,6 +592,22 @@
         "youtube": null
       }
     },
+    {
+      "name": "NepalLink Network",
+      "establishedAt": 1106467200,
+      "location": "Kathmandu, Nepal",
```

Comment:
Add a few location-specific details. Do some research; there is different location information on Google Maps and the official website.

Reply:
so which one should I include then the google location or the website location? And in facebook as welll as in Linkedin the location is Kathmandu, thats why I have added the location Kathmandu, Nepal.
@susanhsrestha

Reply:
Okay, proceed with Kathmandu, Nepal.